bandit/core/cwemap.py

```diff
@@ -23,6 +23,8 @@
     "B505": issue.Cwe.INADEQUATE_ENCRYPTION_STRENGTH,
     "B506": issue.Cwe.IMPROPER_INPUT_VALIDATION,
     "B507": issue.Cwe.IMPROPER_CERT_VALIDATION,
+    "B508": issue.Cwe.CLEARTEXT_TRANSMISSION,
```

Comment:
I didn't get a chance to review the last PR, but I'm wondering if we really need this map here?

Reply:
I was thinking the same, it would make it harder/confusing for 3rd party plugins to have this map here. That being said, I wanted to merge this change and fix up a number of other things as a result of this PR.